Formula/p/percona-server.rb

```diff
@@ -80,6 +79,8 @@ class PerconaServer < Formula
     sha256 "af27e4b82c84f958f91404a9661e999ccd1742f57853978d8baec2f993b51153"
   end
 
+  patch :DATA
```

Comment:
any upstream ref for this patch?

Reply:
No, unfortunately. That was my own workaround for the build error.

Reply:
Could you submit it upstream so we can add that as a ref?